databricks/src/rf_test/train.py

```diff
@@ -0,0 +1,151 @@
+import argparse
+from functools import partial
+
+import mlflow
+import mlflow.sklearn
+
+from cuml.metrics.accuracy import accuracy_score as accuracy_score
+from cuml.preprocessing.model_selection import train_test_split as train_test_split
+from cuml.ensemble import RandomForestClassifier as RandomForestClassifier
+
+from sklearn.model_selection import train_test_split as sk_split
+from sklearn.metrics import accuracy_score as sk_acc
+from sklearn.ensemble import RandomForestClassifier as sk_RF
+import pandas as pd
+
+from hyperopt import fmin, tpe, hp, Trials, STATUS_OK
+
+
+def load_data(fpath, compute_type):
+    """
+    Simple helper function for loading data to be used by CPU/GPU models.
+
+    :param fpath: Path to the data to be ingested
+    :param compute_type: [CPU|GPU]
+    :return: DataFrame wrapping the data at [fpath]. Data will be in either a Pandas or RAPIDS (cuDF) DataFrame
+    """
+    if 'CPU' in compute_type:
```

Comment:
I thought we were going to make this path GPU-only? I think that would be a bit clearer and simplify the example.

Reply:
I'd just removed the CPU path from the notebook. I can take it out of the CLI workflow as well.